src/types/PassageConfig.ts

```diff
@@ -2,9 +2,14 @@ import { FetchAPI } from '../generated';
 import { AuthStrategy } from './AuthStrategy';
 
 export type PassageConfig = {
+    /** The App ID for your Passage Application. */
     appID: string;
```

Comment:
nitpick: a bit out of scope, but could `appId` be added as a replacement for this to keep the naming conventions consistent?

same with https://github.com/passageidentity/passage-node/blob/956d6867ad2ec8c34fca00e4d3fbc02df3904e30/src/classes/Passage.ts#L16 though i don't think it needs to be formally deprecated since it's private

Reply:
I considered it for the PassageConfig object but we'd have to make the new appId field optional and this one optional which just felt like a worse experience than just requiring this change on the major version bump. I did change it for the private field though since that's private anyways

Reply:
gotcha that makes sense